CHANGES.md

```diff
@@ -1,6 +1,21 @@
 
 # Changes
 
+## edge-20.12.2
+
+* Fixed an issue where the `proxy-injector` and `sp-validator` did not refresh
+  their certs automatically when provided externally--like through cert-manager
```

Comment:
is this line supposed to have an emdash?  I think you need to use the html `&mdash;` or an actual emdash character —